src/javascript/app_2/components/elements/data_table.jsx

```diff
@@ -82,4 +83,14 @@ class DataTable extends React.Component {
     }
 }
 
+DataTable.propTypes = {
+    columns         : PropTypes.array,
+    data_source     : PropTypes.array,
+    is_full_width   : PropTypes.bool,
```

Comment:
@jbaek7023 
it's better to define properties in alphabetical order.

Reply:
Ok! working on it now

Reply:
👍